cylc/flow/xtriggers/suite_state.py

```diff
@@ -22,9 +22,9 @@
 import os
 import sqlite3
 
-from cylc.flow.cfgspec.glbl_cfg import glbl_cfg
-from cylc.flow.cycling.util import add_offset
-from cylc.flow.dbstatecheck import CylcSuiteDBChecker
+from cylc.cfgspec.glbl_cfg import glbl_cfg
+from cylc.cycling.util import add_offset
+from cylc.dbstatecheck import CylcSuiteDBChecker
```

Comment:
Hmm, I think these are the old imports from Cylc 7. @ytturi did you install an older version of Cylc? The cylc.flow package imported above is from this repo/branch: https://github.com/cylc/cylc-flow/tree/master/cylc/flow

We released 8.0a1 recently to PYPI, so you should be able to get that version too via pip.

Reply:
I did use the requirements in setup 👍

Reply:
Rollback done